apps/prairielearn/src/ee/institution/admin/course.html.ts

```diff
@@ -0,0 +1,73 @@
+import { html } from '@prairielearn/html';
+import { renderEjs } from '@prairielearn/html-ejs';
+import { type Institution, type Course, type CourseInstance } from '../../../lib/db-types';
+
+export function InstitutionAdminCourse({
+  institution,
+  course,
+  courseInstances,
+  resLocals,
+}: {
+  institution: Institution;
+  course: Course;
+  courseInstances: CourseInstance[];
+  resLocals: Record<string, any>;
+}) {
+  return html`
+    <!DOCTYPE html>
+    <html lang="en">
+      <head>
+        ${renderEjs(__filename, "<%- include('../../../pages/partials/head')%>", {
+          ...resLocals,
+          navPage: 'institution_admin',
+          pageTitle: 'Courses',
+        })}
+      </head>
+      <body>
+        ${renderEjs(__filename, "<%- include('../../../pages/partials/navbar') %>", {
+          ...resLocals,
+          institution,
+          navbarType: 'institution',
+          navPage: 'institution_admin',
+          navSubPage: 'courses',
+        })}
```

Comment:
This little snippet gets really tedious copying around 😓 I wonder if we should have some notion of layout components that can handle this boilerplatey stuff?

Reply:
I'm on board, but I don't see how it will save much code? I'm mainly interested in getting rid of the EJS but that will take some time.

Reply:
Yeah, maybe a good stopgap would be to make a TypeScript component wrapper around the existing EJS?

Reply:
I'm totally fine with that!